src/components/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import "./styles.css";
+
+const defaultValue = "";
+
+class SearchForm extends Component {
+  state = {
+    defaultValue: defaultValue,
+    onSearch: (value) => {},
+  };
+
+  constructor(props) {
+    super(props);
+    this.state.defaultValue = props.defaultValue || defaultValue;
+    this.state.onSearch = props.onSearch;
+  }
+
+  search = () => {
+    let searchInput = document.getElementById("searchInput");
+    this.state.onSearch(searchInput.value);
+  };
+
+  render() {
+    return (
+      <>
```

Comment:
You can remove Fragments here. https://ru.reactjs.org/docs/fragments.html

Reply:
Fixed in every component (d1751c51277a946462dc0a1948c8bc82d9ee253a)